src/Sql/dbo/Stored Procedures/CipherDetails_Create.sql

```diff
@@ -15,7 +15,9 @@
     @ViewPassword BIT, -- not used
     @OrganizationUseTotp BIT, -- not used
     @DeletedDate DATETIME2(7),
-    @Reprompt TINYINT
+    @Reprompt TINYINT,
+    @Key VARCHAR(MAX),
+    @ForceKeyRotation BIT
```

Comment:
⚠️ I realize it's not here for this or other modified procs, but you need to set default values for all these parameters (in case it's called by code that doesn't provide them).

Please do this for all procs (and therefore also your migration script).